lib/commands/sdk.js

```diff
@@ -446,6 +446,11 @@ SdkSubcommands.select = {
 					logger.error(__('Invalid Titanium SDK "%s"', value));
 					return false;
 				}
+				if(parseInt(value.split('.').slice(0,3).join('')) < 320) {
```

Comment:
You should use node-appc to do your version comparison:

```javascript
if (appc.version.lt(value, '3.2.0')) {
```

Also, make sure you have a space between `if` and the `(`. `if` is not a function.